lib/runner/extensions/event.command.js

```diff
@@ -194,6 +177,27 @@ module.exports = {
                         !assertion.passed && assertionFailed.push(assertion.name);
                     });
 
+                var run = this;
+
+                this.host.on('execution.request', function (cursor, id, requestId, request) {
+                    // clone cursor and add source
+                    var cursorWithSource = _.clone(cursor);
+
+                    cursorWithSource.source = 'script'; // @todo - get script type from the sandbox
```

Comment:
Needs better naming. I could think of `cursor.context` meaning if the context is `collection` or a `script`, but we have used `context` in other places to make it misleading.